cinema/views.py

```diff
@@ -43,7 +43,6 @@ def get_serializer_class(self):
 
 class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
-    serializer_class = MovieSessionSerializer
```

Comment:
ok, you've decided to remove this attr.. Another solution was on line 54 write `return self.serializer_class`